libs/cri-storing-service/src/main/java/uk/gov/di/ipv/core/library/cristoringservice/CriStoringService.java

```diff
@@ -120,7 +127,9 @@ public void storeVcs(
                     List.of(vc.serialize()), govukSigninJourneyId, ipAddress);
 
             verifiableCredentialService.persistUserCredentials(vc, criId, userId);
+            vcReceivedThisSession.add(vc.serialize());
```

Comment:
QQ - above line 125-126 - Why we need 2 calls to cimitService   (ciMitService.submitVC & ciMitService.submitMitigatingVcList) ?

Reply:
So the first call is to submit the new VC so that any new CIs it contains can be stored.
The second call is to submit the same VC to CiMits mitigations endpoint - that's where we send VC that _could_ help mitigate a users CIs. 
They could really have been the same endpoint, but it was designed separately for reasons I don't know.

We could in theory only call that second endpoint when we know the user has a mitigating VC;  when they're on a mitigations journey and come back from DCMAW for example. But for ease sake, we decided to just send everything we get to the second endpoint too. CiMit will just ignore anything that isn't actually a mitigating VC.

Reply:
Got it....in those cimit service lambdas are called putContraIndicators & postMitigations.
Yeah sure postMitigations could have handled both as we are calling them together and passing the same payload.